.github/workflows/ci-build.yml

```
@@ -72,9 +72,19 @@ jobs:
           name: build-log-java-${{ matrix.java }}-${{ matrix.os }}
           path: build.log
 
+      - name: Change Files
+        run: |
+          echo modification >> CONTRIBUTING.md
+          touch new-file
+          mkdir new-dir
+          touch new-dir/new-file
+          rm README.md
+
       - name: Verify Changed Files
-        uses: tj-actions/verify-changed-files@v20
         id: verify-changed-files
+        run: |
+          changed_files=$(git diff --name-only ${{ github.sha }} | tr '\n' ' ')
```

```suggestion
          changed_files=$(git diff --name-only ${{ github.sha }}; git ls-files --others --exclude-standard | tr '\n' ' ')
```

Can you try this please?

---

Seems to only fail when files are added but not when files are modified/deleted?

---

Strange... It works on my local bash. May need some more parentheses....